### PR TITLE
Field manual spelling fixes

### PR DESCRIPTION
**Pull request by MaliceGFS**
**When merged this pull request will:**
- Fix several spelling mistakes that are present throughout the field manual in english.
- Tweaks a few sentences or words to make a little more sense.

### CODE REVIEW

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 1]
```diff
@@ -3872,26 +3872,26 @@
 			Du kannst Module auf mehrere Objekte oder Einheiten anwenden, indem du die Module auf eine Einheit platzierst oder in der Nähe, um sie auf mehrere Einheiten anzuwenden.
 		</German>
                 <English>
-			For modules applied on objects or units it is often the case that you can either place it on one unit or place it elsewhere to apply it to multiple units.
+			For modules that apply to objects or units it is often the case that you can either place it on one unit or place it elsewhere to apply it to multiple units.
```
Comment (oOKexOo) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: I apply the module to objects.
The module is applied to objects (by me).
Hence, for modules that *are* applied to objects ...

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 2]
```diff
@@ -3872,26 +3872,26 @@
...
+                <English>Selection Options</English>
                 <French>L'option de choix</French>
                 <Russian>Варианты выбора</Russian>
                 <German>Auswahl Option</German>
             </Key>
             <Key ID="STR_AMAE_SELECTION_OPTION_DESCRIPTION">
                 <Original>
-			Some modules have the option to select objects on which it is apply to.
+			Some modules have the option to select objects to which it's effect(s) will apply to.
 			%1Whenever you hear this sound you can select objects and submit the selection by pressing %3[ENTER]%4.
```
Comment (oOKexOo) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: to which *its* effects will *be* applied to

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 3]
```diff
@@ -3824,7 +3824,7 @@
             </Key>
             <Key ID="STR_AMAE_PLACING_MODULES_DESCRIPTION">
                 <Original>
-			The Ares Modules have informative icons in the module tree.%1The most common are described here:
+			The achilles modules have informative icons in the module tree.%1The most common are described here:
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Achilles should be capitalized

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 4]
```diff
@@ -3852,7 +3852,7 @@
 			%1&lt;img image='\achilles\data_f_achilles\icons\icon_position.paa'/&gt; Dieses Modul wird auf der Position angewendet, auf der es platziert wurde.
 		</German>
                 <English>
-			The Ares Modules have informative icons in the module tree.%1The most common are described here:
+			The achilles modules have informative icons in the module tree.%1The most common are described here:
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: again

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 5]
```diff
@@ -3863,7 +3863,7 @@
             </Key>
             <Key ID="STR_AMAE_PLACING_MODULES_TIP">
                 <Original>
-			For modules applied on objects or units it is often the case that you can either place it on one unit or place it elsewhere to apply it to multiple units.
+			For modules that apply to objects or units it is often the case that you can either place it on one unit or place it elsewhere to apply it to multiple units.
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: I'm going to turn on my pedantic mode on and there is a missing comma between `units, it is`.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml (hunk 1 above)
Comment (CreepPork) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Comma again. And @oOKexOo you would like to think so but in Mr English's mind this is not correct (grammatically).

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml (hunk 2 above)
Comment (CreepPork) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: @oOKexOo nope :D

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 6]
```diff
@@ -4013,10 +4013,10 @@
             </Key>
             <Key ID="STR_AMAE_ARES_FIELD_MANUAL_DESCRIPTION">
                 <Original>
-			Open the %3Field Manual%4 available in the escape menu for useful informations.
+			Open the %3Field Manual%4 available in the escape menu for useful information.
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Maybe Escape should be capitalized as it's a name, no?

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml (hunk 1 above)
Comment (MaliceGFS) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: It actually might be now that I think of it in that given context, I don't know.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml (hunk 6 above)
Comment (MaliceGFS) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: No, not that i'm aware of, it's not a name in the sense of something that needs to be capitalized.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml (hunk 2 above)
Comment (MaliceGFS) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: I wrote it as effect(s) in order to be more explicit and take into consideration a module might have more than one effect on something.

Comment (oOKexOo) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: I'm fine with effect(s). I just forgot you brackets when I wrote it. However, it is still "*its* effect(s)". The apostroph makes no sense here.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml (hunk 1 above)
Comment (oOKexOo) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: I'm actually no longer sure myself. Let's just leave it.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml (hunk 2 above)
Comment (MaliceGFS) [reply @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Is it better now?

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 7]
```diff
@@ -3980,7 +3980,7 @@
             </Key>
             <Key ID="STR_AMAE_EXECUTE_CODE_DESCRIPTION">
                 <Original>
-			The execute code module is only aviable if the server allows clients using it!
+			The execute code module is only available if the server allows for it's use by clients!
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: its

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 8]
```diff
@@ -3998,7 +3998,7 @@
 			%1%2%3_this select 1%4   OBJECT:     gibt ObjNull oder das Objekt, auf dem das Modul platziert wurde, zurück.
 		</German>
                 <English>
-			The execute code module is only aviable if the server allows clients using it!
+			The execute code module is only available if the server allows for it's use by clients!
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: its

**Pull request merged**